petstore/src/main/scala/io/finch/petstore/endpoint.scala

```diff
@@ -0,0 +1,15 @@
+package io.finch.petstore
+
+import io.finch.petstore.model._
+import io.finch.petstore.service._
+import io.finch.petstoreDemo.AuthRequest
+
+object endpoint{
+  val getAllTags: Endpoint[AuthRequest, Seq[String]] = Get / "allTags" / long /> GetAllTags
```

Comment:
Just to recap our conversation yesterday, it'd be more REST-y to use `"tag"` here, and if you're getting all tags you don't need a `long` parameter in the URL. I think it would be best to stick to the Swagger example's API for now, though, and it doesn't expose a resource for tags.